backend/routes/index.js

```diff
@@ -1,7 +1,16 @@
 const express = require( 'express' );
 const moment = require( 'moment' );
+const _ = require( 'lodash' );
 const { CalendarClient } = require( '../app/calendar-client' );
 
+const assertPresent = ( val ) => {
+   if ( typeof val === 'undefined' || val == null || val === '' ) {
```

Comment:
If you don't accept other falsy values like `0` then simple `if(!val)` will do the same job. [Mozilla docs](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Reply:
Fixed in #46 (method removed).